config/default/oidc_auto_login.settings.yml

```diff
@@ -1,5 +1,7 @@
-enabled: true
-client: nomensa
+_core:
+  default_config_hash: 0e6ehRzUiSqmn3UDZTPGqihtSWxeIvhwXy-VLg6i-fo
+enabled: false
+client: client
```

Comment:
You sure about this?

Reply:
No, thats just what `cex` gave me. Thought I had checked all other config files but looks like I missed this one, will remove

Reply:
Possibly a candidate for config_split, but then again...

Certainly not a time to fiddle with config split at the mo.

Reply:
removed